src/index.ts

```diff
@@ -61,6 +67,22 @@ export function form<Decl extends FormDeclaration<any>>(
     }: Sources<Decl>): Sinks<Decl> {
         const touchedKeys = new Set<keyof Decl>();
 
+        let customSubmission$: Stream<KeyboardEvent> = Stream.empty();
+        if (options.customSubmission) {
+            const { customSubmission } = options;
+            const submissionFields = (customSubmission && customSubmission.fields) || undefined;
+            const predicate = (customSubmission && customSubmission.predicate) || defaultPredicate;
+            customSubmission$ = submissionFields
```

Comment:
I didn't get what `submissionFields` exactly means.
In which case you want to configure custom submission?

Perhaps you don't want to give an `undefined` when you have `predicate`?
Or should it be allowed to pass `null` explicitly?
Should an empty set allowed here?

Reply:
@ryota-ka  `submissionFields` defines which fileds in the form can submit itself by custom keybind. If `submissionFIelds` is set to undefined, every fields in the form can be submitted by custom keybind.

Reply:
But, I think it would be better if I change the type definition of `customSubmission` from:

```
customSubmission?: {
            fields?: Array<keyof Decl>;
            predicator?: (e: KeyboardEvent) => boolean;
        };
```
to:

```
customSubmission?: {
            fields: Array<keyof Decl>; // All fileds should be specified explicitly
            predicator: (e: KeyboardEvent) => boolean;
        };
```

Reply:
Refactored: https://github.com/herp-inc/cyclic-form/pull/2/commits/51fae7a1d5ae0bbf59c92699229fce161a71d974